src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045.enums.dfdl.xsd

```diff
@@ -550,7 +550,58 @@ SOFTWARE.
     </restriction>
   </simpleType>
 
-  <simpleType name="messagePrecedenceEnum" dfdlx:repType="tns:enum3">
+  <group name="h_messagePrecedenceCodesGroup">
+    <choice dfdl:choiceDispatchKey="{ $tns:versionSpec }">
+      <!--
+      DFDL has no way to discriminate a choice at unparse time
+      based on data values in the infoset. So choices really cannot
+      be in hidden groups unless the first branch is acceptable as a default.
+
+      Hence, this group cannot be hidden.
+      -->
+      <element name="value_C" dfdl:choiceBranchKey="C"
```

Comment:
Is this name going to conflict with the use of vmf_hdr_c below for h_mesageStandardVersionVMFGroup?

If it's not at the same level of the structure, so won't conflict it would be nice to use the same name in both places for this concept of needing to differentate an enum for C vs. D1. 

If it's going to conflict, then I guess there's nothing to be done about it and thi sname will do fine.

Reply:
The above element is under the element message_precedence_codes so I don't think it conflicts with any other name. 

Like this: 
```
  <message_precedence_codes>
              <value_D>OK_Immediate</value_D>
              <value>OK_Immediate</value>
</message_precedence_codes>
```

Reply:
You mean use vmf_hdr_C or vmf_hdr_D instead of value_C or value_D. Like this: 

```
<message_precedence_codes>
              <vmf_hdr_C>OK_Immediate</vmf_hdr_C>
              <value>OK_Immediate</value>
</message_precedence_codes>
```